src/gwt/src/org/rstudio/core/client/widget/BottomScrollPanel.java

```diff
@@ -73,7 +80,16 @@ public void onResize()
    @Override
    public void scrollToBottom()
    {
-      DomUtils.scrollToBottom(getElement());
+      scrolling_ = true;
+      try
+      {
+         DomUtils.scrollToBottom(getElement());
```

Comment:
None at all; this is just for safety WRT the guard var `scrolling_`. If that code does throw an exception (and it's JSNI, so hard to make guarantees), `scrolling_` will get stuck at `true` and the panel will stop processing scroll events forever.